server/sonar-db-dao/src/main/java/org/sonar/db/component/BranchDto.java

```diff
@@ -115,13 +115,6 @@ public BranchDto setIsMain(boolean isMain) {
     return this;
   }
 
-  /**
-   * This is the getter used by MyBatis mapper.
-   */
-  private String getKee() {
```

Comment:
These methods are in fact used by MyBatis, the mapping will fail if they are removed.

Reply:
will apply suppression so we can fix this together.